controller/internal/exporterset/provisioners/cuttlefish/cuttlefish.go

```
@@ -0,0 +1,480 @@
+}
+
+func encodeConfig(driver virtualtargetv1alpha1.DriverConfig, config map[string]interface{}) (virtualtargetv1alpha1.DriverConfig, error) {
+	raw, err := json.Marshal(config)
+	if err != nil {
+		return driver, fmt.Errorf("marshal driver config: %w", err)
+	}
+	driver.Config = &apiextensionsv1.JSON{Raw: raw}
+	return driver, nil
+}
+
+func resolveDefaultBuild(parameters map[string]interface{}) (string, error) {
```

golangci-lint (2.13.2)

[error] resolveDefaultBuild - result 1 (error) is always nil

(unparam)